lib/mixins/dir-mixin.js

```diff
@@ -136,7 +136,7 @@ export const DirMixin = dedupingMixin((base) => {
     constructor() {
       super();
       /** @type {boolean} */
-      this.__autoDirOptOut = false;
+          this.__autoDirOptOut;
```

Comment:
Weird indent

Reply:
Thanks. 

Could you also check my closure types?  I sort of just spitballed them in there. Am I correct that removing the initial value means we need to put some sort of type on there to keep closure happy?